src/gui/nickdataviewctrl.cpp

```diff
@@ -120,7 +120,6 @@ int NickDataViewCtrl::GetUsersCount() const
 
 void NickDataViewCtrl::DoUsersFilter()
 {
-
```

Comment:
revert please, this creates useless merge conflicts

Reply:
lost by this, i don't recall a marge conflict